### PR TITLE
Custom scrollbar

### LINKED ISSUE #17: Custom scrollbar
Adding a custom scrollbar would improve the look of the website.

### PR DESCRIPTION
**Pull request by KarishmaVanwari**
Fixed #17

### CODE REVIEW

File: src/components/App/App.css [hunk 1]
```diff
@@ -1,124 +1,146 @@
...
-    left: 45%;
-    animation-delay: .2s
-}
-.shadow:nth-child(5){
-    left:auto;
-    right:15%;
-    animation-delay: .3s;
-}
\ No newline at end of file
+::-webkit-scrollbar-thumb {
```
Comment (bislara) [inline src/components/App/App.css:143]: Could you please explain what is this thumb and above track do? I am not aware of it.

**Review by bislara — APPROVED** [review]
Good job. The color looks good too. 🔥🎉🚀

File: src/components/App/App.css (hunk 1 above)
Comment (KarishmaVanwari) [reply src/components/App/App.css:143]: thumb refers to the scrollbar that is visible(in colour that goes up and down) and track refers to the total path that the scrollbar is gonna travel. It thereby creates pseudo element of scrollbar. I hope its clear! @bislara

Comment (KarishmaVanwari) [reply src/components/App/App.css:143]: thumb refers to the scrollbar that is visible(in colour that goes up and down) and track refers to the total path that the scrollbar is gonna travel. It thereby creates pseudo element of scrollbar. I hope its clear! @bislara

**Comment by KarishmaVanwari** [comment]
> Good job. The color looks good too. 🔥🎉🚀

Glad to know that. Thanks!

File: src/components/App/App.css (hunk 1 above)
Comment (bislara) [reply src/components/App/App.css:143]: Oh okay. Noicee. 🚀🙌

**Comment by bislara** [comment]
@all-contributors please add @KarishmaVanwari for code

**Comment by KarishmaVanwari** [comment]
> @all-contributors please add @KarishmaVanwari for code

Thanks :)

**Pull request merged**